test/px-data-table-test.js

```diff
@@ -795,8 +796,14 @@ document.addEventListener("WebComponentsReady", function() {
   remoteDataFilteringFixture2 = document.getElementById('remoteDataFiltering2');
   remoteDataFilteringFixture2.tableData = minidata;
 
-  // greedyHeightWithScroll = document.getElementById('greedyHeightWithScroll');
-  // greedyHeightWithScroll.tableData = minidata;
+  greedyHeightWithScrollFixture = document.getElementById('greedyHeightWithScroll');
+  greedyHeightWithScrollFixture.tableData = minidata;
+
+  clientSmartRowSelectionWithLargeHeightFixture = document.getElementById('clientSmartRowSelectionWithLargeHeight');
```

Comment:
Can all these references to `clientSmartRowSelectionWithLargeHeight` and `serverSmartRowSelectionWithLargeHeightFixture` be deleted (lines 1, and 802-806)